server/datastore/mysql/migrations/tables/20250922083056_AddTableMDMAndroidProfiles.go

```diff
@@ -136,6 +135,28 @@ ALTER TABLE mdm_configuration_profile_labels
 		return fmt.Errorf("alter mdm_configuration_profile_labels table: %w", err)
 	}
 
+	// our mysql version at the time this constraint was added did not support CHECK constraints so this may or may not exist for us
+	// to delete, so we create the new wider constraint above then, optionally, delete the older narrow one
```

Comment:
Wow that's a great catch and something to keep in mind... I knew mysql 5 did not enforce those, I hadn't realized it didn't even create them.